docs/generated/devkit/NxJsonConfiguration.md

```diff
@@ -31,6 +31,7 @@ Nx.json configuration
 - [neverConnectToCloud](../../devkit/documents/NxJsonConfiguration#neverconnecttocloud): boolean
 - [nxCloudAccessToken](../../devkit/documents/NxJsonConfiguration#nxcloudaccesstoken): string
 - [nxCloudEncryptionKey](../../devkit/documents/NxJsonConfiguration#nxcloudencryptionkey): string
+- [nxCloudId](../../devkit/documents/NxJsonConfiguration#nxcloudid): string
```

Comment:
Spoke with Isaac, it's not a huge deal to leave these in the devkit docs for now. The main docs for nx-cloud login will come with an upcoming PR.